app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetController.scala

```diff
@@ -292,18 +269,18 @@ class EditSetController @Inject() (
       (editSetRecord, transformer) => transformer(editSetRecord)
     )
 
-  private def generateEditSetView(id: String, user: User, editSetReorder: EditSetReorder)(implicit
+  private def generateEditSetView(id: String, user: User, ordering: RowOrdering)(implicit
     request: Request[AnyContent]
   ): Result = {
     logger.info(s"The edit set id is $id ")
     val currentEditSet = editSets.getEditSet()
 
-    val editSetEntries = currentEditSet.entries.sorted(getSorter(editSetReorder))
+    val editSetEntries = currentEditSet.entries.sorted(ordering.ordering)
```

Comment:
If we rename `RowOrdering` to `EditSetEntryRowOrder` and the field to `editSetEntryRowOrder` and if we then rename the `val` in the renamed class to `currentOrdering` then this would become `editSetEntryRowOrder.currentOrdering` which would be a bit more meaningful to me than `ordering.ordering` :)

Reply:
ha, indeed... names are hard. Will change.

Reply:
Agreed. Agreed.

Reply:
Much better. Thank you.